knowledge-layout/src/main/java/dev/ikm/komet/layout/component/version/field/KlComponentSetField.java

```diff
@@ -14,11 +13,9 @@
  * manage and interact with a collection of versioned component entities
  * in the context of knowledge layout.
  *
- * @param <E> The type of the entity in the set.
- * @param <V> The type of the entity version.
  * @TODO should we have separate list and set types, or should they be combined into one component that can be constrained to prohibit addition of duplicates?
  *
  */
 @RegularName("Component Set Field")
-public interface KlComponentSetField<E extends Entity<V>, V extends EntityVersion> extends KlSetField<Set<E>> {
+public interface KlComponentSetField extends KlField<Set<EntityProxy>> {
```

Comment:
Can we change the generic type definition to `KlField<IntIdSet>`?

Reply:
I believe we can make use of `IntIdSet` like the following: 

`IntIdSet newSet = existingIntIdSet.with(someNid);`

I believe the objective is to make the Custom control respond to drag and drop and populating using nids instead of publicIds and/or EntityProxies.